### ISSUE TITLE
runcommand.io links are broken in readme

### ISSUE THREAD

**Issue by tomjn** [wp-cli/doctor-command#159]
The readme of this git repo relies on links to runcommand.io, but this site does not exist and leads to a web host placeholder page

**Comment by schlessera** [comment]
Yes, the site is dead by now. I recently overhauled the `profile-command`, and I will do the same here once I get to it.

Thanks for the report!

**Comment by cjhaas** [comment]
Any chance we could get this back on someone's radar? If the documentation is somewhere else online I'd be glad to try merging it into the readme or something else if that would help, too.

**Comment by igorradovanov** [comment]
I have the same question. Is there a mirror site with the docs?

**Comment by danielbachhuber** [comment]
Here are the corresponding links on my site:
* https://danielbachhuber.com/tip/doctor-default-checks/
* https://danielbachhuber.com/tip/customize-doctor-config/

Maybe this should live in the handbook, though?

**Comment by danielbachhuber** [comment]
> Maybe this should live in the handbook, though?

Started moving the docs with https://github.com/wp-cli/handbook/pull/429

Once they are synced into the handbook, we can merge https://github.com/wp-cli/doctor-command/pull/174

I also need to:
* [ ] Port over the secondary docs to the handbook.
* [ ] Set up redirects from my site to the handbook.
